ReactiveLocation/ReactiveLocation.swift

```diff
@@ -90,30 +91,31 @@ internal extension CLLocationManager {
 
 public class ReactiveLocation: ReactiveLocationService {
 
+    private lazy var manager = LocationManagerConfigureBlock()
+    
     public init() { }
 
-    static public func locationProducer(_ managerFactory: LocationManagerConfigureBlock? = nil) -> SignalProducer<CLLocation, LocationError> {
-        let manager = LocationManagerConfigureBlock()
+    public func locationProducer(_ managerFactory: LocationManagerConfigureBlock? = nil) -> SignalProducer<CLLocation, LocationError> {
+        let manager = self.manager
         managerFactory?(manager)
 
         guard let delegateObject = manager.delegateObject else { return SignalProducer.empty }
-        return merge([errorSignal(delegateObject), locationSignal(delegateObject)])
+        return SignalProducer.merge([errorSignal(delegateObject), locationSignal(delegateObject)])
             .on(started: {
                 manager.startUpdatingLocation()
                 }, terminated: {
                 manager.stopUpdatingLocation()
         })
     }
 
-    static public func singleLocationProducer(_ managerFactory: LocationManagerConfigureBlock? = nil) -> SignalProducer<CLLocation, LocationError> {
-
-        let manager = LocationManagerConfigureBlock()
+    public func singleLocationProducer(_ managerFactory: LocationManagerConfigureBlock? = nil) -> SignalProducer<CLLocation, LocationError> {
+        let manager = self.manager
         managerFactory?(manager)
 
         guard let delegateObject = manager.delegateObject else { return SignalProducer.empty }
 
         // -requestLocation guarantees callback called once
-        return merge([errorSignal(delegateObject), locationSignal(delegateObject)]).take(first: 1)
+        return SignalProducer.merge([errorSignal(delegateObject), locationSignal(delegateObject)]).take(first: 1)
             .on(started: {
                 if #available(iOS 9, *) {
```

Comment:
Shouldn't we just bump the project target version to 9.0 and drop iOS 8 support?

Reply:
I don't think it's necessary just because of this.